pkgs/tools/security/hash-identifier/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, fetchFromGitLab, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "hash-identifier";
+  version = "1.2";
+
+  src = fetchFromGitLab {
+    owner = "kalilinux";
+    repo = "packages/hash-identifier";
+    rev = "20ebdd2f33ffc1bdcc010ff582be203696b0bb2e";
+    sha256 = "2cGF7lRXG+w7QsztbEWanvifPtjOwPNMvm660z4rxZA=";
+  };
+
+  installPhase = ''
+    patchShebangs --host ./hash-id.py
+    mv ./hash-id.py ./hash-identifier
+    mkdir -p $out/bin
+    install -m775 -t $out/bin hash-identifier
```

Comment:
Just use install for this.

Reply:
See https://github.com/NixOS/nixpkgs/pull/127490#discussion_r654963819 whole discussion.